### ISSUE TITLE
Cannot insert any text

### ISSUE THREAD

**Issue by Muzza** [slynch8/10x#30]
With today's version, if I try to type anything half way through a word, the cursor immediately jumps to the end of the word.
I cannot insert any text basically.

**Comment by slynch8** [comment]
I can't reproduce this. Can you try clearing your settings and see if it goes away?

**Comment by Muzza** [comment]
Yes it still does it if I delete the settings and restart 10x.
I noticed it only happens when it does not pop-up an autocomplete box.

Here is a minimal example
```
typedef struct MyStruct
{
	int	aaa[4];
}
MyStruct;

void Func( MyStruct* p )
{
	int i = p->aaa[ 0 ];
}
```

Place your cursor after -> and before aaa. Type 'z'. The cursor inserts z then moves to the end of aaa.

**Comment by slynch8** [comment]
fixed (0.21.11.0)
